### ISSUE TITLE
"Missing Awesome badge": False positive?

### ISSUE THREAD

**Issue by nschloe** [sindresorhus/awesome-lint#23]
awesome-lint reports
```
✖  1:1  Missing Awesome badge after the main heading  awesome/badge
```
for [my list](https://github.com/nschloe/awesome-numerical-software/) which starts off with
```markdown
# Awesome Numerical Software [![Awesome](https://awesome.re/badge-flat.svg)](https://github.com/sindresorhus/awesome)
```
not sure what's wrong here.

**Comment by nschloe** [comment]
Aha, the link must point to https://awesome.re (fixed by myself) and the list cannot be `-flat`. Can probably called a bug.

**Comment by transitive-bullshit** [comment]
Agreed, this will be a very common false signal (false negative or false positive depending on how you look at it lol).

I propose we do the following for `awesome/badge`:
- For both the image src and link href, accept a set of whitelisted values including common ones found in current awesome lists like @nschloe has above.
- Add some basic regex for `https://cdn.rawgit.com/sindresorhus/awesome/.*/media/badge.svg` or something similar

Above all else, we want to avoid false negatives (e.g., invalid errors).
